### ISSUE TITLE
[BITV] 9.4.1.2/4.4 - "Photos settings" Modal Dialog - The dialog has not been marked as such with the appropriate role of "dialog".

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/photos#1686]
The implementation also lacks the aria-modal="true" attribute. With the value "true", browsers and screen readers will be informed that the content contained in the dialog window is foreground and that other content on the web page should be ignored. More information can be found at https://www.hellbusch.de/dialogfenster-für-screemreader-als-modalkennzeichnen and https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html. (1)

- [ ] Fix
- [ ] Backport

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v4-n4

</details>

**Comment by Fenn-CS** [comment]
This does not seem to be accurate anymore. "Photos Settings" dialog uses : [`NcAppSettingsDialog`](https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/NcAppSettingsDialog/NcAppSettingsDialog.vue) which in turn, returns [`NcModal`](https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/NcModal/NcModal.vue) which has the right aria markers as shown on

https://github.com/nextcloud/nextcloud-vue/blob/4142020e0d0c00bfd4804aae86192e05db465c4a/src/components/NcModal/NcModal.vue#L180-L184

**Comment by AndyScherzinger** [comment]
Then let's close it as fixed 👍 Thanks @Fenn-CS
